packages/onr-common/src/services/Http.ts

```diff
@@ -0,0 +1,185 @@
+import { stringify } from 'qs';
+import Axios, { AxiosResponse, AxiosError } from 'axios';
+import getConfig from 'next/config';
+
+declare type RequestMethods =
```

Comment:
use axios `method`

Reply:
just refactoring here. but planning to rewrite this part anyway